Change some GUI strings to address SD card vs. flash memory ambiguity

DE-cr opened the pull request:
With reference to https://github.com/fredlcore/BSB-LAN/issues/632#issuecomment-1999089017

Moved storage location config to general section, as it applies for both logging and webserver file system.
**Please note: I don't know if this requires additional measures to address possible shifts in EEPROM storage locations!**

Changed some GUI strings to address SD card vs. flash memory ambiguity.

Removed "ESP32" from GUI string for energy saving, as other ESP32 only strings don't have that prefix either (OTA, SD/flash).

Adjusted whitespace in one code area, so that comments properly align regardless of tab width setting.

fredlcore commented:
Please move the storage settings back to their original place as this is where it thematically belongs best. While it also affects the webserver, hardly any configuration file that I received for debugging purposes had this feature enabled.

DE-cr commented:
Done. New change description is then:

~~Moved storage location config to general section, as it applies for both logging and webserver file system.
**Please note: I don't know if this requires additional measures to address possible shifts in EEPROM storage locations!**~~

Changed some GUI strings to address SD card vs. flash memory ambiguity.

Removed "ESP32" from GUI string for energy saving, as other ESP32 only strings don't have that prefix either (OTA, SD/flash).

Adjusted whitespace in one code area, so that comments properly align regardless of tab width setting.

DE-cr commented:
Btw, I use the webserver functionality, but only to store the javascript libraries required for /DG locally. This just fyi.

fredlcore commented on BSB_LAN/BSB_LAN_EEPROMconfig.h:
```diff
@@ -206,9 +206,9 @@ const configuration_struct config[]={
   {CF_TRUSTEDIPADDRESS, 2, CCAT_IPV4,     CPI_TEXT,      CDT_IPV4,           OPT_FL_ADVANCED, CF_TRUSTEDIPADDRESS_TXT, sizeof(trusted_ip_addr)}, //immediately apply
   {CF_TRUSTEDIPADDRESS2,2, CCAT_IPV4,     CPI_TEXT,      CDT_IPV4,           OPT_FL_ADVANCED, CF_TRUSTEDIPADDRESS_TXT, sizeof(trusted_ip_addr2)},//immediately apply
-#ifdef ESP32
+  #ifdef ESP32
```
Indention was correct before. It does not follow code-level indention, but compiler-directive-level indention, and there is no #if clause before

DE-cr replied:
Sorry, when moving the config setting back to its original place, I copied the indentation for the preprocessor directives from line 179 ("When in Rome, do as the Romans."). I'll fix both locations now.

fredlcore commented:
Thanks!
